src/main/java/hudson/scm/CVSSCM.java

```diff
@@ -438,6 +440,9 @@ private class RepositoryBrowser {
         // we don't provide a way for users to edit this, other than by manually editing their XML config
         private String changelogEncoding = "UTF-8";
         
+        @SuppressFBWarnings(
+            value = "MC_OVERRIDABLE_METHOD_CALL_IN_CONSTRUCTOR",
+            justification = "Will fix later on")
```

Comment:
This is a common suppression in Jenkins.
```suggestion
            justification = "Jenkins relies on calling overridable methods from constructors")
```